data_diff/dbt_parser.py

```diff
@@ -144,6 +144,7 @@ def set_connection(self):
                 "warehouse": credentials.get("warehouse"),
                 "role": credentials.get("role"),
                 "schema": credentials.get("schema"),
+                "insecure_mode": credentials.get("insecure_mode") or False,
```

Comment:
**nit**: `credentials.get("insecure_mode", False)

Reply:
ty simplified this